vernfongchao/feature/About/redesign

vernfongchao opened the pull request:
Redesign About Page

Add Picture for About

Add styled component under Section

Move Affiliates and Org List as components

vernfongchao commented:
@ri0nardo @zoobot Randy wants to use to font Montserrat as the font for these new redesigned pages however its not built into css. How did we want to go about this? importing it from google fonts or downloading the font into the project itself?

vernfongchao commented:
@ri0nardo @mwpark2014 @zoobot let me know if there's anything I should change

ri0nardo commented:
@vernfongchao can you add screenshots?

vernfongchao commented:
@ri0nardo first comment

ri0nardo commented:
@vernfongchao i can send you a markup if you want, but theyre all minor changes

1. on the very top there isn't a title "about us", needs to be removed.
2. The font style seems to be all regular, when the section titles are bold or semi-bold.
3. The "join our slack" green button font style needs to be bold. If the style goes has no padding, please add.
4. On mobile, where the section title text begins, there needs to be a margin from the navigation bar. 

Other than that the core of the work looks great.

vernfongchao commented:
> * on the very top there isn't a title "about us", needs to be removed.
> * The font style seems to be all regular, when the section titles are bold or semi-bold.
> * The "join our slack" green button font style needs to be bold. If the style goes has no padding, please add.
> * On mobile, where the section title text begins, there needs to be a margin from the navigation bar.
@ri0nardo 
1. removed about us header
2. Its actually bolded, without bold it looks like this
![image](https://user-images.githubusercontent.com/91238232/213582569-50905f50-aae8-4893-937c-37ce7645707a.png)
3. changed font to bold and added padding
4. There is a margin, you just don't see it because its using the new header and not the old one, the new header is smaller

ri0nardo commented:
@vernfongchao if thats the bold then thats fine. Its just that the bold on Figma is significantly fatter text. If its bold then its approved.
![Capture](https://user-images.githubusercontent.com/98299952/213586433-d21e7b2f-40e8-4b32-8daf-a30585626e26.PNG)

mwpark2014 commented:
One more minor comment: what are the dimensions of the photo in this PR? Probably completely fine to merge in a high res photo in this PR, but we might want to be smart in serving high-res vs low-res photos in the future (the about page is pretty low prio with highest prio being the photo gallery feature in the future).

vernfongchao commented on client/src/pages/About/About.js:
```diff
@@ -1,62 +1,170 @@
+            </p>
+          </div>
+          <div className="about__main__section__numsection">
+            <div className="about__main__section__numsection__relative">
```
The reason why im having an absolute nesting inside of a relative is mainly for structure. The relative div is used to give structure but also control to its children divs. Its actually a well known technique used by alot of people.

https://css-tricks.com/absolute-positioning-inside-relative-positioning/

mwpark2014 replied:
Hmm, so the relative element gives structuring for the absolute element child. But there is just one child that doesn't have any positioning fields like `top`, `left`, etc, so I'm not sure why you need this extra div element. Couldn't you put these fields into the parent `&__relative` element? I could be wrong!

```
 &__absolute {
        position: absolute;
        width: 100%;
        display: flex;
        justify-content: center;
        color: #3fab4530;

        span {
          font-size: 15.5rem;
        }
      }
```

vernfongchao commented on client/src/pages/About/About.scss:
```diff
@@ -1,44 +1,175 @@
+
+        &__relative {
+          position: relative;
+          width: 10;
```
Yeah that was a mistake, was migrating things over from styled components.
